src/grpc/infer_handler.cc

```diff
@@ -1100,6 +1131,8 @@ ModelInferHandler::InferResponseComplete(
   if (response_created) {
     delete response;
   }
+
+  delete response_release_payload;
```

Comment:
@tanmayv25 can you confirm that the state is going to be reused and thus we should not associate anything that ties to a particular inference to the state lifecycle? This looks right to me if that is the case.

Reply:
This is accurate understanding.